### PR TITLE
Fix a couple of missing types

### PR DESCRIPTION
**Pull request by alexwlchan**

### CODE REVIEW

**Review by rcantin-w — DISMISSED** [review]
👍

File: common/data/hardcoded-ids.ts [hunk 1]
```diff
@@ -3,26 +3,12 @@
...
-  café: {
-    id: 'WsuZKh8AAOG_NyUo',
-    name: 'Café',
-  },
-  readingRoom: {
-    id: 'Wvlk4yAAAB8A3ufp',
-  },
+export const collectionVenueIds = {
+  galleries: 'Wsttgx8AAJeSNmJ4',
+  libraries: 'Library',
```
Comment (rcantin-w) [inline common/data/hardcoded-ids.ts:8]: Should this be `WsuS_R8AACS1Nwlx`?

Comment (alexwlchan) [reply common/data/hardcoded-ids.ts:8]: yup, oops

**Comment by rcantin-w** [comment]
Screenshot 2023-01-16 at 09 48 29

The pending UI change is "Galleries" changing to "Galleries and reading room" which looks a bit off. I can't quite place why it's happening, though, as that data seems to be coming from Prismic. Maybe it was changed for a little bit, then changed again but Chromatic hasn't rerun?

**Comment by alexwlchan** [comment]
It’s because we used to get the venue name from our hard-coded list, then I switched to get it from Prismic, which has the full title.

I’m tired of fighting this particular issue though, so I’ve removed it from this PR for now.

**Comment by rcantin-w** [comment]
I'm sure we could figure out a workaround if you want to discuss it later on!

**Review by rcantin-w — APPROVED** [review]

**Pull request merged**